src/array_api_stubs/_draft/searching_functions.py

```diff
@@ -137,6 +146,132 @@ def searchsorted(
     """
 
 
+def top_k(
+    x: array,
+    k: int,
+    /,
+    *,
+    axis: Optional[int] = None,
+    mode: Literal["largest", "smallest"] = "largest",
+) -> Tuple[array, array]:
+    """
+    Returns the ``k`` largest (or smallest) elements of an input array ``x`` along a specified dimension.
```

Comment:
```suggestion
    Returns the values and indices of the ``k`` largest (or smallest) elements of an input array ``x`` along a specified dimension.
```
Otherwise it would be identical to the description of `top_k_values`.